index.html

```diff
@@ -1068,6 +1070,26 @@ <h3 class="title">Ryan Juang</h3>
 
     </section>
 
+    <section id="codeofconduct" aria-labelledby="codeofconductLabel">
+
+        <div class="wrapper">
+
+            <div class="row">
+
+                <div class="col-xs-12">
+
+                    <h3 id="codeofconductLabel">Code of conduct</h3>
+
+                    <p>Our conference is dedicated to providing a harassment-free conference experience for everyone, regardless of gender, age, sexual orientation, disability, physical appearance, body size, race, or religion (or lack thereof). We do not tolerate harassment of conference participants in any form. Sexual language and imagery is not appropriate for any conference venue, including talks, workshops, parties, Twitter and other online media. Conference participants violating these rules may be sanctioned or expelled from the conference without a refund at the discretion of the conference organisers.</p>
```

Comment:
Haha, yes - it was copied from nodeconf.eu. I can sprinkle in some `z`s if you like :)